crates/uv/src/commands/tool/run.rs

```diff
@@ -49,8 +50,13 @@ pub(crate) async fn run(
     cache: &Cache,
     printer: Printer,
 ) -> Result<ExitStatus> {
+    let uv_command_to_run = if invoked_via_uvx {
+        "uvx"
+    } else {
+        "uv tool run"
+    };
```

Comment:
Maybe instead of a `invoked_via_uvx` bool we should have an enum that implements `Display`? Seems slightly better to have a type. Doesn't feel critical though, I'm not sure what else we'd do with it.

Reply:
Yeah enum with type seems nice :)

Reply:
@blueraft interested in adding a `ToolRunCommand` enum and using that?

Reply:
Done

Reply:
Thanks!